phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/ContainerInformationTab.java

```diff
@@ -110,11 +111,23 @@ private void populate() {
         Button deleteButton = new Button(tr("Delete container"));
         deleteButton.setOnMouseClicked(event -> this.onDeletePrefix.accept(container));
 
-        informationPane.getChildren().addAll(informationContentPane, spacer, /* changeEngineComboBox, */ deleteButton);
+        Region buttonSpacer = new Region();
+        buttonSpacer.setPrefHeight(20);
```

Comment:
This looks like something we want to move to css/less

Reply:
Yes, but not in this pr (it's the same like the other spacer above).